### PR TITLE
Rename korath to korath exiles

### PR DESCRIPTION
**Pull request by jafdy**
This should rename the exiles after the player is told about them in Wanderers: Kor Efret 4. I'm confident I have caught all the cases after that point where they appear. 
I haven't had a chance to check whether the fleet governments can be changed the way I just did.

### CODE REVIEW

**Comment by gunqqer** [comment]
Huh, never noticed this. I guess it was because all other korath and Kor sesteo, mérito, efrit, etc. And thy are the only Korath

**Comment by Bladewood** [comment]
The reason they're not called the Korath Exiles is because we get to see them from the start of the game.

This implementation should work, but I don't really agree with it.

**Comment by Elyssaen** [comment]
Does this change fleet presences in Korath Exile systems?

**Comment by KiLEdEnNis** [comment]
thing is when you first encounter them you may not know they are exiles, just after you met the Efreti and heard the stories

**Comment by gunqqer** [comment]
I'm personally fine with them just being named Korath. They are the original Korath who got exiled because they wouldn't join the Efrit.

**Comment by jafdy** [comment]
@Elyssaen No. Or at least if it dose then I've done something wrong.

Of my 3 PRs this is the least significant as it has no real effect on anything. In truth I am not strongly in favor of it although I think it is a good idea (or I wouldn't have PRed it).

**Comment by gunqqer** [comment]
I am personally opposed to this. The only living Korath alive are the exiles and the Efrit (as of now, later updates may change this). The Sestor and Meriti are reminents from the two factions that fought a civil war. By nature Korath are violent and power hungry, just like the Wanderers are not. The exiles are the 'true' Korath and thus should be named so

**Comment by jafdy** [comment]
That's like the argument of if the normal Hai or the Unfettered Hai are the "true" Hai.

**Comment by tehhowch** [comment]
I guess this ties into the dynamic of "how do system government labels arise?" E.g. Why are the Hai systems labeled as "Unfettered Hai / Hai" rather than "Hai / Fettered Hai"? I think this is from how the inhabitants see themselves: the inhabitants of the 3 northern systems *distinguish themselves* from the rest of the Hai systems by claiming to be "Unfettered", while the rest of the Hai systems' inhabitants view themselves as just "Hai", rather than believing that they have been behaviorally neutered and thus seeing themselves as "Fettered Hai".

So for Korath systems, the exiled Korath view themselves as they are - Korath. The Efreti have gone out of their way to become friendly, and named themselves as such, ergo Kor Efret. The automata factions have likewise distinguished themselves from the rest of Korath systems.

With that view, then I would not support this PR either.

**Comment by gunqqer** [comment]
The Efreti are not the original Korath. They changed. They are peaceful now. @tehhowch put it very well. The exiles are the embodiment of the original Korath. The normal Hai are the original Hai.

**Comment by Bladewood** [comment]
The Efreti are composed of two groups (or three, depending on your perspective):

1. The Korath who originally did not wish to take part in the Civil War.

2. The Kor Sestor and Kor Mereti remnants who agreed to the Drak's deal.

The Korath belligerents who did not agree to the deal were exiled to the Core.

Whether or not anyone actually accepted the Drak's deal is unknown.

File: data/wanderers middle.txt [hunk 1]
```diff
@@ -752,6 +752,39 @@ mission "Wanderers: Kor Efret 3"
 
 
 
+event "Rename Korath Exiles"
```
Comment (Elyssaen) [inline data/wanderers middle.txt]: I'd add:

`	"reputation: Korath Exiles" = "reputation: Korath"`

**Comment by Elyssaen** [comment]
Just checked it out and had a look. Does what it says, which is good, but as per my review comment I think you should synchronise the reputation values. Doesn't make sense that my -6000 Korath reputation disappears once the player chooses to identify them differently.

My question about fleet presences was because I hadn't seen your PR *does* changes the government of the Korath fleets that appear in their systems. So the answer was yes, it does change it, and it's good that it does.

I don't have a strong opinion for or against this. I think I lean *slightly* against. One reason is that depending on how the next part of the Wanderers campaign goes, it's possible they'll stop being exiles very soon. And renaming them again would then feel messy to me. I don't feel like just "Korath" is a problem, ultimately.

**Comment by jafdy** [comment]
They are linked. For every 1 rep you lose with the "Korath" you lose 1 rep with the "Korath Exiles". This will only take effect after this is introduced but that is a minor problem because there is no way to increases rep with the "Korath" with a mission before the change happens.
Will that line you propose actually work? The wiki says that conditions can be set to a value but not a to a different condition (or the same condition even).

**Comment by Elyssaen** [comment]
The line does work, I tested it out to be sure.

The Wiki is out-of-date, you've been able to set a condition to another condition since https://github.com/endless-sky/endless-sky/pull/2383 was merged a few weeks ago.

**Comment by Amazinite** [comment]
@jafdy They'll only be linked for new saves.

**Comment by Bladewood** [comment]
I don't think it matters, because they're always hostile to you (until that one part in the Wanderer campaign, but that completely resets your reputation anyway).

**Comment by gunqqer** [comment]
wish we could get @endless-sky to comment on this as it is a question of lore.

**Comment by endless-sky** [comment]
This really doesn't seem necessary - I think a player is as likely to be confused about why the name of their government changed, as anything else.

**Pull request closed without merging**